tools/Microsoft.StandardsPolice/StandardsPoliceCompileModule.cs

```diff
@@ -71,8 +71,8 @@ internal static void ScanSyntaxTree(IList<Diagnostic> diagnostics, SyntaxTree sy
                     acceptableOrder = true;
                 }
                 if (!acceptableOrder &&
-                    priorUsingDirective.Name.ToString().StartsWith("System.") &&
-                    !usingDirective.Name.ToString().StartsWith("System."))
+                    priorUsingDirective.Name.ToString().StartsWith("System") &&
```

Comment:
This avoids some false negatives.

    (priorUsingDirective.Name.ToString() == "System" ||
    priorUsingDirective.Name.ToString().StartsWith("System.")) &&
    !usingDirective.Name.ToString().StartsWith("System."))

which flags:

    using System2;
    using MyNamespace;